fe/fe-core/src/main/java/org/apache/doris/nereids/stats/ExpressionEstimation.java

```diff
@@ -121,11 +123,23 @@ public class ExpressionEstimation extends ExpressionVisitor<ColumnStatistic, Sta
      * returned columnStat is newly created or a copy of stats
      */
     public static ColumnStatistic estimate(Expression expression, Statistics stats) {
-        ColumnStatistic columnStatistic = expression.accept(INSTANCE, stats);
-        if (columnStatistic == null) {
-            return ColumnStatistic.UNKNOWN;
+        try {
+            ColumnStatistic columnStatistic = expression.accept(INSTANCE, stats);
+            if (columnStatistic == null) {
+                return ColumnStatistic.UNKNOWN;
```

Comment:
why here not use withAvgSizeByte?

Reply:
this is copied from original codes.
yes, it is better to set size.